<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecycleBin</class>
 <widget class="QWidget" name="RecycleBin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 251, 255);</string>
  </property>
  <widget class="QLabel" name="lb_file">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>121</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(102, 102, 102, 1)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;文件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_createtime">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>121</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(102, 102, 102, 1)</string>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_operation">
   <property name="geometry">
    <rect>
     <x>1331</x>
     <y>121</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(102, 102, 102, 1)</string>
   </property>
   <property name="text">
    <string>操作</string>
   </property>
  </widget>
  <widget class="PictureBox" name="recyclebin_pic" native="true">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>320</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_no_content">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>690</y>
     <width>167</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>无近期删除文件</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contentList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>1400</width>
     <height>711</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1440</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(193, 220, 245, 1);
background-color: rgba(193, 220, 245, 1)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_titile" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>100</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>38</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 650 12pt &quot;Microsoft JhengHei&quot;;
</string>
    </property>
    <property name="text">
     <string>回收站</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>23</y>
      <width>495</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32769</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>搜索瑞记</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>contentList</zorder>
  <zorder>lb_file</zorder>
  <zorder>lb_createtime</zorder>
  <zorder>lb_operation</zorder>
  <zorder>recyclebin_pic</zorder>
  <zorder>lb_no_content</zorder>
  <zorder>line_2</zorder>
  <zorder>widget_titile</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
